src/Resources/public/js/app/components/braintree-method-component.js

```diff
@@ -33,10 +33,10 @@ define(function (require) {
         nonceInput: null,
 
         initialize: function (options) {
-            this.options = _.extend({}, this.options, options);
+            this.options = $.extend({}, this.options, options);
             this.$el = this.options._sourceElement;
```

Comment:
I know this line hasn't changed, but it's standing out to me.. I don't think `this.options._sourceElement` returns a jQuery object does it? If not, then could we please update the variable name to simply be `this.el`, so we avoid any confusing in the future.
(Having `$` before a variable is a standard way of indicating that a variable contains a jQuery object, therefore all of those helpers are available)

Reply:
Interesting!